src/test/java/org/commcare/formplayer/auth/AuthTestUtils.java

```diff
@@ -0,0 +1,49 @@
+package org.commcare.formplayer.auth;
+
+import static org.commcare.formplayer.util.Constants.PART_ANSWER;
+import static org.commcare.formplayer.util.Constants.PART_FILE;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.multipart;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+
+import org.commcare.formplayer.util.Constants;
+import org.commcare.formplayer.utils.FileUtils;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.mock.web.MockPart;
+import org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Common methods for session auth and hmac auth tests
+ */
+public class AuthTestUtils {
+
+    /**
+     * Use the 'clear_user_data' endpoint for 'full auth' which required user details.
+     */
+    public static MockHttpServletRequestBuilder getFullAuthRequestBuilder(String body) {
+        return post(String.format("/%s", Constants.URL_CLEAR_USER_DATA))
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(body)
+                .with(SecurityMockMvcRequestPostProcessors.csrf());
+    }
+
+    /**
+     * Use the 'answer_media' endpoint for 'full auth' with multipart
```

Comment:
I think this comment is wrong. This specifically creates a request for the `mock_mulipart_request` endpoint right?